.travis.yml

```diff
@@ -31,10 +31,12 @@ before_install:
   - travis_retry composer self-update
 
 install:
-  - travis_retry composer require --dev --no-update squizlabs/php_codesniffer
   - if [ "$deps" != low ]; then travis_retry composer update --no-interaction --prefer-dist; fi;
   - if [ "$deps" == low ]; then travis_retry composer update --no-interaction --prefer-dist --prefer-lowest; fi;
 
 script:
-  - vendor/bin/phpcs
-  - vendor/bin/phpunit --coverage-text --exclude-group integration
+  - vendor/bin/phpunit --coverage-clover=coverage.clover --exclude-group integration
+
+after_build:
```

Comment:
"after_build" does not exist. See here: https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle

You want to use: `after_script` or (preferable) `after_success`.

Reply:
haha I blindly wrote what @stof told me IRL :)
And I explicitly want `after_script` because scrutinizer recommands to always send coverage info even when tests fails (see the tip [there](https://scrutinizer-ci.com/docs/tools/external-code-coverage/#changes-in-third-party-configuration))